src/command_modules/azure-cli-policyinsights/azure/cli/command_modules/policyinsights/_params.py

```diff
@@ -0,0 +1,113 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.parameters import resource_group_name_type
+
+from azure.cli.command_modules.resource._completers import (
+    get_policy_set_completion_list, get_policy_completion_list,
+    get_policy_assignment_completion_list, get_providers_completion_list, get_resource_types_completion_list)
+
+from ._validators import validate_resource
+
+
+def load_arguments(self, _):
+    for scope in ['state', 'event']:
+        with self.argument_context('policy {}'.format(scope)) as c:
+            c.argument(
+                'management_group_name',
+                options_list=['--management-group-name', '-m'],
+                arg_group='Scope',
+                help='Management group name.')
+            c.argument(
+                'resource_group_name',
+                arg_type=resource_group_name_type,
+                arg_group='Scope')
+            c.argument(
+                'resource',
+                options_list=['--resource', '-r'],
```

Comment:
the `-r` option should not be given as it is not available for other instances of `--resource` in the CLI and the default option is relatively short

Reply:
Forgot to remove. Will do.